meshmode/interop/firedrake/connection.py

```diff
@@ -726,7 +726,7 @@ def build_connection_from_firedrake(actx, fdrake_fspace, grp_factory=None,
     # a numpy array
 
     # Get the reordering fd->mm.
-    flip_mat = get_simplex_element_flip_matrix(ufl_elt.degree(),
+    flip_mat, _perm = get_simplex_element_flip_matrix(ufl_elt.degree(),
                                                fd_unit_nodes)
```

Comment:
```suggestion
    flip_mat, _perm = get_simplex_element_flip_matrix(ufl_elt.degree(),
                                                      fd_unit_nodes)
```
? The one below is also not aligned anymore.